### PR TITLE
fix TFProcessLocalWorker with callbacks

### PR DESCRIPTION
**Pull request by estebanlm**
we need some tweaks for using TFProcessLocalWorker with callbacks

### CODE REVIEW

File: src/UnifiedFFI/LibC.class.st [hunk 1]
```diff
@@ -69,6 +69,13 @@ LibC >> fgetc: stream [
 	 ^self ffiCall: #(int* fgetc(void* stream))
 ]
 
+{ #category : #'api - accessing' }
+LibC >> getpid [
+	"Get character from stream."
```
Comment (gcotelli) [inline src/UnifiedFFI/LibC.class.st]: Is this comment right? I looks unrelated to the FFI call

**Review by MarcusDenker — APPROVED** [review]
